Tasks/AzureRmWebAppDeploymentV4/task.loc.json

```diff
@@ -225,7 +225,7 @@
       "type": "pickList",
       "label": "ms-resource:loc.input.label.RuntimeStackFunction",
       "defaultValue": "",
-      "required": true,
+      "required": false,
```

Comment:
please handle the null case in the task runtime.

Reply:
good catch. made required changes.